src/main/java/io/dev/jobprep/domain/security/oauth/presentation/OauthController.java

```diff
@@ -1,39 +1,40 @@
 package io.dev.jobprep.domain.security.oauth.presentation;
 
 
-import com.auth0.jwt.interfaces.DecodedJWT;
+import io.dev.jobprep.domain.security.jwt.application.AuthService;
 import io.dev.jobprep.domain.security.jwt.application.dto.TokenInfo;
 import io.dev.jobprep.domain.security.oauth.domain.PrincipalDetails;
-import io.dev.jobprep.domain.security.jwt.application.JwtService;
-import io.dev.jobprep.domain.security.oauth.application.PrincipalDetailsService;
 import io.dev.jobprep.common.swagger.template.OauthSwagger;
 import io.dev.jobprep.domain.security.oauth.presentation.dto.TokenResponse;
+import jakarta.servlet.http.HttpServletResponse;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/api/v1/oauth2")
 @RequiredArgsConstructor
 public class OauthController implements OauthSwagger{
-    private final JwtService jwtService;
-    private final PrincipalDetailsService principalDetailsService;
+
+    private final AuthService authService;
+
     // 토큰 재발급
-    @PostMapping("/reissue")
-    public ResponseEntity<TokenResponse> refreshToken(@RequestHeader(value = "xRefreshToken") String refreshToken) {
-        jwtService.isTokenValid(refreshToken);
-        DecodedJWT decodeJWT = jwtService.verifyToken(refreshToken);
-        String userId = jwtService.extractUserId(decodeJWT);
-
-        PrincipalDetails principalDetails = (PrincipalDetails) principalDetailsService.loadUserByUsername(userId);
-
-        //새로운 토큰 페어 생성
-        TokenInfo tokenInfo = jwtService.generateTokenInfo(userId, principalDetails.getEmail(), principalDetails.getUserRoles());
-        TokenResponse tokenResponse = TokenResponse.from(tokenInfo);
-        return ResponseEntity.ok()
-                .body(tokenResponse);
-    }
+    @GetMapping("/reissue")
+    public ResponseEntity<Void> reissue(@CookieValue(value = "XRefreshTokens") String refreshToken,
```

Comment:
value가 복수형이어야 하는 하는 이유가 있을까요? 예를 들면, 일반적으로 복수형으로 사용된다거나,,?

```suggestion
    public ResponseEntity<Void> reissue(@CookieValue(value = TokenHeaderConstants.REFRESH_HEADER) String refreshToken,
```

Reply:
아앗 바꿔야겠네요